Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -6,20 +6,77 @@ import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 
 import {
   boundPointOnPlane,
-  getAssociatedLinesName,
   rotateVector,
   updateState,
+  getOtherLineName,
+  getLinePlaneName,
+  getLineInPlaneName,
 } from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers';
 
 import {
   ScrollingMethods,
   InteractionMethodsName,
+  planeNameToViewType,
+  lineNames,
 } from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/Constants';
 
 export default function widgetBehavior(publicAPI, model) {
   model._isDragging = false;
   let isScrolling = false;
 
+  // FIXME: label information should be accessible from activeState instead of parent state.
```

Comment:
Is this FIXME (and the one below) actionable in this PR?

Reply:
No, this is not related to this PR. It would be something to do in the future (and to discuss first)